kas-bootstrap/kas_boostrap.go

```diff
@@ -54,6 +60,18 @@ func run(ctx context.Context, opts Options) error {
 		return fmt.Errorf("failed to create client: %w", err)
 	}
 
+	// Create a client for the management cluster using the mounted service account.
+	managementCfg, err := rest.InClusterConfig()
+	if err != nil {
+		return fmt.Errorf("failed to get management cluster config from service account: %w", err)
+	}
+	managementClient, err := client.New(managementCfg, client.Options{Scheme: configScheme})
+	if err != nil {
+		return fmt.Errorf("failed to create management cluster client: %w", err)
+	}
+
+	logger.Info("Successfully created management cluster client")
+
 	// This binary is meant to run next to the KAS container within the same pod.
 	// We briefly poll here to retry on race and transient network issues.
 	// This to avoid unncessary restarts of this container.
```

Comment:
While we are making changes to this file, can you update the spelling here so codespell doesn't complain.
```suggestion
	// This to avoid unnecessary restarts of this container.
```